applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -298,22 +326,33 @@ static int handle_at_sleep(const char *at_cmd, enum shutdown_modes *mode)
 		}
 		if (shutdown_mode == SHUTDOWN_MODE_IDLE) {
 			slm_at_host_uninit();
-			enter_idle();
+			enter_idle(true);
 			*mode = SHUTDOWN_MODE_IDLE;
 			ret = 0; /*Will send no "OK"*/
 		} else if (shutdown_mode == SHUTDOWN_MODE_SLEEP) {
 			slm_at_host_uninit();
-			enter_sleep(true);
+			enter_sleep(false);
```

Comment:
Was that change intended? The commit message does not mention anything about changing the behavior of `AT#XSLEEP=1`.

Reply:
Thanks a lot for this, don't know why I messed up here.